src/main.ts

```diff
@@ -127,7 +129,7 @@ async function notifyTeams(notificationMessage: NotificationMessage) {
 }
 
 checkBasicInputsSet();
-if (core.getInput('job_status').toUpperCase() !== 'CANCELLED') {
+if (core.getInput('job_status').toUpperCase() !== jobStatus.cancelled.status) {
```

Comment:
you are upper casing the job_status but your constant value is lower case so it seems like they will never be equal.